t/cli/test_etcd.sh

```diff
@@ -80,6 +80,24 @@ fi
 
 echo "passed: properly handle the error when connecting to etcd without auth"
 
+# Check etcd retry if connect failed
+git checkout conf/config.yaml
+
+echo '
+etcd:
+  host:
```

Comment:
Better to expand the tab to space

Reply:
> Better to expand the tab to space

Sorry, I‘m not understand.
I just copy other code in same file and do some modify. 
The code style is same. And it's seems no tab in this file.